towhee/engine/execution/vectorized_execution.py

```diff
@@ -28,9 +25,17 @@ def __vcall__(self, *arg, **kws):
         if hasattr(self._op, '__vcall__'):
             return self._op.__vcall__(*arg, **kws)
         elif len(arg) == 1:
-            return [self._op(x) for x in arg[0]]
+            res = [self._op(x) for x in arg[0]]
+            try:
```

Comment:
why we need a try-catch here?

Reply:
For multi output cases, we need to turn [(o1, o2), (o3, o4) ..] to ([o1, o3, ...], [o2, o4, ...])